yarn-project/aztec/src/bin/index.ts

```diff
@@ -38,7 +38,9 @@ async function main() {
     networkValue = args[networkIndex].split('=')[1] || args[networkIndex + 1];
   }
 
-  await enrichEnvironmentWithChainConfig(getActiveNetworkName(networkValue));
+  const networkName = getActiveNetworkName(networkValue);
+  await enrichEnvironmentWithChainConfig(networkName);
+  await enrichEnvironmentWithNetworkConfig(networkName);
```

Comment:
I'm not sure which one should take priority? I could make an argument either way

Reply:
I think probably the network config.